etcdserver/config.go

```diff
@@ -160,6 +166,9 @@ func (c *ServerConfig) print(initial bool) {
 		plog.Infof("initial advertise peer URLs = %s", c.PeerURLs)
 		plog.Infof("initial cluster = %s", c.InitialPeerURLsMap)
 	}
+	if !c.PeerTLSInfo.Empty() {
```

Comment:
can we still print this out in etcd.go? we need to keep this consistent with client tlslinfo.